README.md

```diff
@@ -458,6 +458,10 @@ to Miri failing to detect cases of undefined behavior in a program.
   This is much less likely with Stacked Borrows.
   Using Tree Borrows currently implies `-Zmiri-strict-provenance` because integer-to-pointer
   casts are not supported in this mode, but that may change in the future.
+* `-Zmiri-tree-borrows-no-precise-interior-mut` makes Tree Borrows
+  track interior mutable data on the node-level instead of on the
+  byte-level as is done by default.  Therefore, with this flag, Tree
+  Borrows will be more permissive.
```

Comment:
```suggestion
* `-Zmiri-tree-borrows-no-precise-interior-mut` makes Tree Borrows
  track interior mutable data on the level of references instead of on the
  byte-level as is done by default.  Therefore, with this flag, Tree
  Borrows will be more permissive.
```